avr-asm.tmLanguage

```diff
@@ -137,7 +168,7 @@
 			<key>comment</key>
 			<string>Dec number constant</string>
 			<key>match</key>
-			<string>\b(0|[1-9]\d+)\b</string>
+			<string>\-?\b(\d+)\b</string>
```

Comment:
Don't try to directly edit *.tmLanguage file. Your changes will be lost since the file is generated from *.JSON-tmLanguage one.